CONTRIBUTING.md

```diff
@@ -1 +1,37 @@
+# Contribution Guidelines
+
+## Auto-Formatting
+ECL uses clang-format to auto-format the code. Currently is is using the clang-format-6.0.
+The enforced style is based on the google style. Even tough not really followed at the moment, Google's [Style Guide](https://google.github.io/styleguide/cppguide.html) is the place to look for advise.
```

Comment:
I'm not sure about the meaning of "Even tough not really followed at the moment"

'advise' should be 'advice'